src/status_im/contexts/wallet/common/account_switcher/view.cljs

```diff
@@ -26,7 +26,8 @@
            switcher-type       :account-options
            type                :no-title}}]
   (let [{:keys [color emoji watch-only?]} (rf/sub [:wallet/current-viewing-account])
-        networks                          (rf/sub [:wallet/selected-network-details])]
+        networks                          (rf/sub [:wallet/selected-network-details])
+        sending-collectible?              (rf/sub [:wallet/sending-collectible?])]
```

Comment:
Sub used to display the account switcher when the user is NOT sending a collectible